### ISSUE TITLE
Alexnet’s based model performance in pytorch

### ISSUE THREAD

**Issue by tung-qle** [thuml/CDAN#12]
Hi. Thanks for your excellent work. I am re-implementing your result for my project. I have an issue: when the base CNN is AlexNet, do you get the same performance on Office 31 data set on Pytorch as your model on Caffe. 
I try to run your Pytorch code for the task Amazon to Webcam (anh use AlexNet) but get around 71% accuracy after more than 25000 iterations (which is not as reported in the paper). It is worth noticing that your Resnet version works perfectly fine on my computer.

**Comment by tung-qle** [comment]
The Alexnet model in my run is taken from here: https://drive.google.com/uc?export=download&id=0B-7I62GOSnZ8NzVxZndDU2dYcHM

**Comment by hehai131** [comment]
Yes, I have confused the same issue long times in DA, I have tried many ways by using pytorch but still have no idea. 

I think the problem mainly lies in the pre-trained alexnet model. Sincerely hope that someone can give tips for pytorch version alexnet code.

**Comment by tung-qle** [comment]
I try to re implement this model on Tensorflow and get the same result as Pytorch version as well. Is there a possibility that the problem with difference of inner mechanisms across different platforms because I believe the checkpoint of Pytorch has the same statistical result as ones publiced by the original Alexnet ...

**Comment by sy565612345** [comment]
Hi,
The results on AlexNet were from caffe and the implementations and pretrained models of AlexNet in caffe and pytorch are different, which may cause the issue.

**Comment by tung-qle** [comment]
> Hi,
> The results on AlexNet were from caffe and the implementations and pretrained models of AlexNet in caffe and pytorch are different, which may cause the issue.

Thanks for the answer. However, I am afraid that is not the case. The author used the original AlexNet (as implemented in the network.py file), not the implementation and pretrained model of torchvision, which are indeed different from Caffe's.
